src/Formatter/FormatterServiceProvider.php

```diff
@@ -24,7 +25,7 @@ public function register()
             return new Formatter(
                 new Repository($container->make('cache.filestore')),
                 $container->make('events'),
-                $this->app['flarum']->storagePath().'/formatter'
+                $this->app[Paths::class]->storage.'/formatter'
```

Comment:
Why not $container->make? Also not insisting on anything, moreso curious as to the difference.

Reply:
I probably didn't bother changing it, because that's what the old code used. :thinking: 

I typically prefer `make()` when I write new code, because that one is part of the `Container` *contract*, while the array offset stuff comes from the concrete `Container` class.

Reply:
Agreed, these should use the `make()` method instead. Good spot Sasha.
```suggestion
                $this->app->make(Paths::class)->storage.'/formatter'
```

Reply:
I don't think this change belongs here. I agree with the proposal, but if we want to do it, we should do it for all occurrences (and this PR doesn't touch all of them). Feel free to take care of that in a separate PR. :wink: